buildscripts/checkstyle.xml

```diff
@@ -52,7 +52,7 @@
   <module name="TreeWalker">
     <module name="RegexpSinglelineJava">
       <property name="format"
-                value="(?&lt;!import static io.opentelemetry.sdk.testing.assertj.)OpenTelemetryAssertions\."/>
+                value="(?&lt;!import (?:static )?io.opentelemetry.sdk.testing.assertj.)OpenTelemetryAssertions\.(?![A-Z]\w*[;\s]*)"/>
```

Comment:
The regex is too strict and catching inner classes, which in this case can't be statically imported. I updated the pattern to exclude both regular and static imports from the package, and to allow uppercase class names while still catching lowercase method calls that should be statically imported.

Reply:
that is quite hard to understand - can we make it 2 assertions?